VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -649,21 +647,29 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
         this.audioRecordingService.cleanupSubscriptions();
     }
 
-    private handleWowzaConnectionState(isConnected: boolean) {
-        if (isConnected) {
-            if (this.audioRecordingService.restartActioned) {
-                this.notificationToastrService.showAudioRecordingRestartSuccess(this.audioRestartCallback.bind(this));
-            }
-            this.continueWithNoRecording = false;
-        } else if (this.audioRecordingService.restartActioned) {
-            this.notificationToastrService.showAudioRecordingRestartFailure(this.audioRestartCallback.bind(this));
+    private handleConnectionState(stateIsConnected: boolean): void {
```

Comment:
similar to the previous comment - might be able to fix the sonar issue by handling the boolean in the calling code, something like

`.subscribe(stateIsConnected => stateIsConnected ? this.onWowzaConnected() : this.onWowzaDisconnected());`

Reply:
ditto